From 9528acb261ae04e3530fea056071b96f62473da8 Mon Sep 17 00:00:00 2001
From: Shreeya1904 <shreeya1904@gmail.com>
Date: Tue, 4 Jun 2024 12:22:43 +0530
Subject: [PATCH] change in rebase.txt

---
 rebase.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rebase.txt b/rebase.txt
index 092f22d..7a6415d 100644
--- a/rebase.txt
+++ b/rebase.txt
@@ -1 +1 @@
-x+y
\ No newline at end of file
+This is to rebase the branch
\ No newline at end of file
-- 
2.41.0.windows.3

